.travis.yml

```diff
@@ -7,10 +9,27 @@ php:
   - 7.0
   - 7.1
   - nightly
-  - hhvm
 
 matrix:
+  fast_finish: true
+  include:
+    - php: hhvm
+      sudo: true
+      dist: trusty
+      group: edge # Use edge image until the next travis CI image update
+      addons:
+        apt:
+          packages:
+            - mysql-server-5.6
+            - mysql-client-core-5.6
+            - mysql-client-5.6
+      services:
+        - mysql
+        - postgresql
+        - redis-server
+      env: CUBRID_VERSION=9.3.0/CUBRID-9.3.0.0206 CUBRID_PDO_VERSION=9.3.0.0001
   allow_failures:
+    - php: hhvm
```

Comment:
@samdark the 2 failures on hhvm are related to pgsql. These could be related to the differences between hhvm's pgsql module and the zend pgsql module, or related to a couple of items that are fixed in upcoming releases for hhvm 3.15.3 and hhvm 3.16.0, or could be related to other php vs hhvm differences.

```php
1) yii\tests\unit\framework\db\pgsql\CommandTest::testSaveSerializedObject
Failed asserting that 0 matches expected 1.
/tests/framework/db/pgsql/CommandTest.php:86
/vendor/phpunit/phpunit/phpunit:47
2) yii\tests\unit\framework\db\pgsql\CommandTest::testBindParamValue
Failed asserting that false is true.
/tests/framework/db/CommandTest.php:199
/vendor/phpunit/phpunit/phpunit:47
```